onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -31,8 +33,13 @@ class Attention : public OpKernel, public AttentionCPUBase {
                                    /*out*/ bool& used_shared_buffers) override;
 
  private:
-  BufferUniquePtr packed_weights_;
-  size_t packed_weights_size_ = 0;
+  BufferUniquePtr q_packed_weights_;
```

Comment:
It might help to defined as  
    BufferUniquePtr packed_weights_[3];
Then you can use qkv_index to access. That could avoid switch statements in code and help performance.

Reply:
@tianleiwu, I believe this can be done at few places.
I missed this comment, will update the code accordingly in my next PR.